Zend/tests/serializable_deprecation.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+Serializable deprecation
+--FILE--
+<?php
+
+interface I extends Serializable {}
+abstract class A implements Serializable {}
+
+class C extends A implements I {
+    public function serialize(): string {}
+    public function unserialize(string $data) {}
+}
+
+class D extends A implements I {
+    public function serialize(): string {}
+    public function unserialize(string $data) {}
+    public function __serialize(): array {}
+    public function __unserialize(array $data) {}
+}
+
+?>
+--EXPECTF--
+Deprecated: The Serializable interface is deprecated. If you need to retain the Serializable interface for cross-version compatibility, you can suppress this warning by implementing __serialize() and __unserialize() in addition, which will take precedence over Serializable in PHP versions that support them in %s on line 6
```

Comment:
```suggestion
Deprecated: The Serializable interface is deprecated. For cross-version compatibility, also implement __serialize() and __unserialize() which take precedente over Serializable in %s on line 6
```

Reply:
```suggestion
Deprecated: The Serializable interface is deprecated. Implementing __serialize() and __unserialize() instead (or in addition, if support for old PHP versions or serialization payloads needs to be retained) in %s on line 6
```

Reply:
I think the main thing I want to convey here is that the alternative is __serialize/__unserialize, and you only need both for supporting multiple versions, otherwise you don't.